src/promptflow/setup.py

```diff
@@ -26,7 +26,7 @@
     "flask>=2.2.3,<3.0.0",  # Serving endpoint requirements
     "dataset>=1.6.0,<2.0.0",  # promptflow.storage
     "sqlalchemy>=1.4.48,<2.0.0",  # sqlite requirements
-    "pandas>=1.5.3,<2.0.0",  # load data requirements
+    "pandas>=2.0.0,<2.1.2",  # load data requirements
```

Comment:
looks like all our tests has passed? we might write >=1.5.3, < 3 here

Reply:
sure, updated.